siriuspy/siriuspy/clientconfigdb/types/as_diagnostics.py

```diff
@@ -1468,6 +1468,198 @@ def get_dict():
     [':TRIGGER7DirPol-Sel', 1, 0.0],
     [':TRIGGER7RcvLen-SP', 1, 0.0],
     [':TRIGGER7TrnLen-SP', 1, 0.0],
+    [':TRIGGER_LAMP0RcvInSel-SP', 0, 0.0],
```

Comment:
wouldn't be better to make a loop here?

Reply:
agreed! I was just replicating the pattern of BPM PVs... but I can change there also

Reply:
done!